dali/test/dali_test_config.cc

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <mutex>
+#include <cstring>
+#include <cassert>
+#include "dali/test/dali_test_config.h"
+
+namespace dali {
+namespace testing {
+namespace program_options {
+
+namespace {
+
+/// Path to Dali_extra repository
+std::string _dali_extra_path;  // NOLINT
```

Comment:
leading underscore - what for?

Reply:
Since there's `dali_extra_path` function, calling `return _dali_extra_path` is required, so that this call is not ambiguous.